scripts/version.sh

```
@@ -0,0 +1,90 @@
+# Parse version components
+MAJOR=$(echo $CURRENT_VERSION | cut -d. -f1)
+MINOR=$(echo $CURRENT_VERSION | cut -d. -f2)
+PATCH=$(echo $CURRENT_VERSION | cut -d. -f3)
+
+echo ""
+echo "Select increment type:"
+echo "1) Patch (+0.01)  - مثال: 1.0.0 -> 1.0.1"
+echo "2) Minor (+0.1)   - مثال: 1.0.0 -> 1.1.0"
+echo "3) Major (+1)     - مثال: 1.0.0 -> 2.0.0"
+echo "4) Custom version"
+read -p "Enter choice (1-4): " choice
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)